tests/e2e/framework/kubernetes.go

```diff
@@ -109,12 +115,6 @@ func (k *KubeInfo) Setup() error {
 	}
 
 	if !*skipSetup {
-		if err = util.CreateNamespace(k.Namespace); err != nil {
```

Comment:
Why are we removing this ? if not cluster wide don't we need to create the namespace anyways ? Also we might want to get @chxchx changes about removing stack driver as this will probably break it.

Reply:
We no longer need to create the ns explicitly, it's done by the istio.yaml.

Reply:
Please keep in mind the only difference in the cluster wide yaml file is in the environment variable passed to istio-ca. Feel free to diff the files.

Reply:
got it